### PR TITLE
split ovn-central and ovn-monitor

### PR DESCRIPTION
**Pull request by hongzhen-ma**

### CODE REVIEW

File: dist/images/install.sh [hunk 1]
```diff
@@ -1953,6 +1810,146 @@ spec:
...
             secretName: kube-ovn-tls
 ---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kube-ovn-monitor
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      OVN components: northd, nb and sb.
```
Comment (oilbeater) [inline dist/images/install.sh:1820]: same as above

File: pkg/ovnmonitor/metric.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ var (
 		prometheus.GaugeOpts{
 			Namespace: metricNamespace,
 			Name:      "ovn_status",
-			Help:      "OVN Health Status. The values are: health(1), unhealth(0).",
+			Help:      "OVN Health Status. The values are: standby(2), health(1), unhealth(0).",
```
Comment (oilbeater) [inline pkg/ovnmonitor/metric.go:11]: health -> active?

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Comment by oilbeater** [comment]
The ovn.yaml and ovn-ha.yaml should also be changed, some user do not use the install.sh scripts and prefer to use yaml directly

File: dist/images/install-pre-1.16.sh [hunk 3]
```diff
@@ -1908,6 +1765,146 @@ spec:
...
             secretName: kube-ovn-tls
 ---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kube-ovn-monitor
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      OVN components: northd, nb and sb.
```
[the remark this answers is not in the document]
Comment (hongzhen-ma) [reply dist/images/install-pre-1.16.sh:1775]: modified

File: dist/images/install.sh (hunk 1 above)
Comment (hongzhen-ma) [reply dist/images/install.sh:1820]: modified

File: pkg/ovnmonitor/metric.go (hunk 2 above)
Comment (hongzhen-ma) [reply pkg/ovnmonitor/metric.go:11]: modified

**Review by oilbeater — APPROVED** [review]

**Pull request merged**